superset-frontend/src/explore/components/controls/DateFilterControl/DateFilterLabel.tsx

```diff
@@ -314,7 +314,7 @@ export default function DateFilterLabel(props: DateFilterControlProps) {
       <Divider />
       <div>
         <div className="section-title">{t('Actual time range')}</div>
-        {validTimeRange && <div>{evalResponse}</div>}
+        {validTimeRange && <div>{t(evalResponse)}</div>}
```

Comment:
Same here... is this something we would expect to have in the translation files, or is it a more dynamic value (as it sounds, based on the name)

Reply:
Yes, it's a date or it's write 'No filter', so I think in fact that will juste translate 'no filter' and don't touch to other result

Reply:
Maybe it's worth putting a little ternary condition or similar in there, e.g. `evalResponse = 'no filter' ? t('no filter') : evalResponse`? Or maybe there's somewhere further upstream in the code that's more appropriate to the the translation?